build.sbt

```diff
@@ -57,21 +58,15 @@ lazy val client = (project in file("clients/java"))
   .settings(
     name := s"$artifactNamePrefix-client",
     commonSettings,
-    libraryDependencies ++= Seq(
-      "com.fasterxml.jackson.core" % "jackson-annotations" % jacksonVersion,
-      "com.fasterxml.jackson.core" % "jackson-core" % jacksonVersion,
-      "com.fasterxml.jackson.core" % "jackson-databind" % jacksonVersion,
-      "com.fasterxml.jackson.datatype" % "jackson-datatype-jsr310" % jacksonVersion,
-      "org.openapitools" % "jackson-databind-nullable" % openApiToolsJacksonBindNullableVersion,
-      "com.google.code.findbugs" % "jsr305" % "3.0.2",
-      "jakarta.annotation" % "jakarta.annotation-api" % "1.3.5" % Provided,
-
-      // Test dependencies
-      "junit" %  "junit" % "4.13.2" % Test,
-      "org.junit.jupiter" % "junit-jupiter" % "5.9.2" % Test,
-      "net.aichler" % "jupiter-interface" % JupiterKeys.jupiterVersion.value % Test,
-    ),
-
+    javaCheckstyleSettings(file("dev") / "checkstyle-config.xml"),
+    libraryDependencies ++=
+      Dependencies.findBugs ++
+      Dependencies.guava ++ 
+      Dependencies.hadoop ++ 
+      Dependencies.jackson(includeAnnotations = true) ++
+      Dependencies.Jakarta.annotation ++ 
+      Dependencies.junit4,
+    libraryDependencies ++= Dependencies.junit5.value,
```

Comment:
Why is this dependency separately added

Reply:
This is becomes Junit5 is not a simple dependency, it's a setting that needs to be evaluated in order by SBT, see the `value` keyboard invoked to retrieve the setting value here . https://github.com/unitycatalog/unitycatalog/pull/37/files#diff-ad2642dc77b3679e4ad2c7d3b2f59a2dcf48c1bf5b93a558a4fa44828315a34cR94-R97

Unfortunately concatenating playing `Seq[ModuleID]` and `Def.Initialize[Seq[ModuleID]`` doesn't work. I haven't tried if we put junit5 as a first element on the list like so

```scala
  libraryDependencies ++= Dependencies.junit5.value ++ Dependencies.others
```

happy to try if you think that's better